pydm/widgets/timeplot.py

```diff
@@ -118,27 +181,69 @@ def resetUpdatesAsynchronously(self):
     def max_x(self):
         return self.data_buffer[0, -1]
 
+    def max_y(self):
```

Comment:
How is this different than `maxY` ? 
This will create a confusion for users since both properties will have the same name only differing by camelCase and Python style.

Reply:
Actually, this method isn't used anymore. I'll remove it.

Reply:
Sounds good.